docs/detections/alerts-runtime-fields.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[alerts-runtime-fields]]
+== Add runtime fields to detection alerts
+
+Runtime fields are fields that you can add to documents after you've ingested your data, and they are evaluated at query time. For example, you could combine two fields and treat them as one, or perform calculations on existing data and use the result as a separate field.
+
+You can add a runtime field to your detection alerts from the *Alerts* table. The new field becomes available on all alerts within {elastic-sec}.
```

Comment:
```suggestion
You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all alerts in {elastic-sec}.
```

Reply:
Slight edit, so it doesn't sound like the new field is only applicable _inside_ the Security app:
```suggestion
You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all {elastic-sec} alerts.
```